src/test/java/com/shop/frontend/pages/ProductPage.java

```diff
@@ -1,11 +1,44 @@
 package com.shop.frontend.pages;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
 
-public class ProductPage {
+import java.util.Collections;
+import java.util.List;
 
+public class ProductPage {
     WebDriver driver;
+    // Author: Priyanka
+    By btn_Shop = By.xpath("/html/body/header/div/div/ul/li[2]/a");
+    By txt_SearchField = By.id("search");
+    By productList = By.className("col");
+
+    public ProductPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    // Author: Priyanka
+    public void clickOnButtonShop() {
+        WebElement element = driver.findElement(btn_Shop);
+        element.click();
+
+    }
+
+    // Author: Priyanka
+    public void clickOnSearchField(String value) {
```

Comment:
This method is called clickOnSearchField. However, it does more then clicking the field. It clicks the searchfield, writes value variable content into it, and then sends an enter command. It does a full search, so perhaps it should be called something similar to "searchFor".

Reply:
Sure, will do that.